services/Dashboard/VideoConference/index.ts

```diff
@@ -0,0 +1,7 @@
+const getRoomToken = (userId: string) => {
+  return {
+    token: process.env.NEXT_PUBLIC_HMS_DEV_TOKEN,
```

Comment:
After creating the ODI account, remind me to share the token to everyone shad.

Reply:
This token will be dynamically generated by our backend in production Alex. This token is just there for testing purposes